docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc

```diff
@@ -0,0 +1,8 @@
+[id='using-standalone-perspectives-intro-con']
+= Using standalone perspectives
+
+{CENTRAL} provides different specialized editors for authoring assets which have different formats. As per an asset's format a specific designer is used for editing it. {CENTRAL} has a feature that allows you to use these editors individually. This feature is known as the standalone perspective mode of the editor or simply the _standalone perspectives_.
+
+As a developer, this provides you with the ability to embed a standalone perspective in your web application and then use it to edit rules, processes, decision tables, and so on. After embedding a perspective you can edit an asset in your own application without switching to {CENTRAL}. You can use this feature to customize your web application. In addition to standalone perspectives you can also embed standalone custom pages (dashboards) in your applications.
+
+You can access a standalone perspective by using a specific web-link in a browser with the `standalone` and `perspective` parameters. A standalone perspective's link may contain additional parameters as well.
```

Comment:
You can access a standalone perspective by using a specific web address.. in a browser with the `standalone` and `perspective` parameters. A standalone perspective's web address may also contain additional parameters.

Just a thought. Up to you.

Reply:
Changed as suggested.